components/progress-bar/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "coherent-gameface-progress-bar",
-  "version": "3.2.0",
+  "version": "4.0.0",
```

Comment:
Why is it a breaking change? What does this change break exactly? Maybe the minor version should be updated instead as this change is a feature and it won't break anything IMO.

Reply:
Oh sorry I didn't check that! It is fine then!